### ISSUE TITLE
Research: Configuration manager

### ISSUE THREAD

**Issue by vlvkobal** [netdata/netdata#6317]
##### Feature idea summary
The idea is to create a centralized service for gathering and distributing configuration across the whole Netdata ecosystem.

Related issues: #2360, #4731, #6350, #5669, #6432

##### Expected behavior
There is a central configuration service

**Comment by FedericoCeratto** [comment]
Also see #5090

**Comment by cakrit** [comment]
Blocked by #6432

**Comment by cakrit** [comment]
Closing in favor of #6432
